wireless/wapi/src/wapi.c

```diff
@@ -913,6 +914,7 @@ static int wapi_reconnect_cmd(int sock, int argc, FAR char **argv)
 static int wapi_save_config_cmd(int sock, int argc, FAR char **argv)
 {
   char essid[WAPI_ESSID_MAX_SIZE + 1];
+  char bssid[INET_ADDRSTRLEN];
```

Comment:
why `bssid` is `INET` and not `ETH`?

Reply:
WiFi use the same MAC as Ethernet

Reply:
Yes. `INET_ADDRSTRLEN` is for IP address, so it is changed to 20.